config/testconfig.js

```diff
@@ -18,12 +17,10 @@ config["welcome"]["channel"] = "unstable_bot";
 config["leagues"]["45+45"]["spreadsheet"]["key"] = "1BeRN76zaB_uCCrCra2yTEEw_r6C5_b8P59aN_BrJsyA";
 config["leagues"]["45+45"]["spreadsheet"]["serviceAccountAuth"] = {
```

Comment:
Can we completely get rid of serviceAccountAuth? I don't think we use the spreadsheets at all anymore, right?